types/react-native/index.d.ts

```diff
@@ -8822,6 +8822,11 @@ export function processColor(color: any): number;
 
 export function __spread(target: any, ...sources: any[]): any;
 
+export interface ErrorUtils {
+    setGlobalHandler: (callback: (error: any, isFatal: boolean | undefined) => void) => void;
+    getGlobalHandler: () => (error: any, isFatal: boolean | undefined) => void;
```

Comment:
Gotcha. Maybe type it as `isFatal?: boolean` instead?

I’m also wondering if it would make sense to extract the handler signature and share that between the two functions?